File: SquidoMVC/Squido/Squido.Mvc/Models/Entities/Author.cs
namespace Squido.Models.Entities;

public class Author
{
    public Guid AuthorId { get; set; }
    public string FullName { get; set; }
    public string? Bio { get; set; }
    public string? ImageUrl { get; set; }
    public ICollection<Book> Books { get; set; }
}

File: SquidoClient/src/types/author.ts
export interface Author {
  id?: string // Add id field as an alternative to authorId
  authorId: string
  fullName: string
  bio?: string
  imageUrl?: string
  isDeleted: boolean
  books: any[]
}

export interface AuthorResponse {
  isSuccess: boolean
  message: string
  errorCode: string | null
  data: Author[]
  exceptionMessage: string | null
}
